### PR TITLE
All user stories and tests completed

### PR DESCRIPTION
**Pull request by charlierdm**

### CODE REVIEW

**Comment by charlierdm** [comment]
- All user stories are included within the code. 
- Each user stories has a corresponding test.
- Given more time I would like to improve the way I used a double as that was a new concept to me and I didn't fully understand it. I'd also like to improve my READme as it is currently unfinished.

**Comment by ianmcnicholas** [comment]
Pull Request:
Commits are infrequent and could probably do with more clarification in their messages.
Succinct in the comments left for the reviewer - good.

README:
Well presented
References user requirements well.
Accept that it is incomplete from the comments - for completion it would be good to see information on how to run the program and what methods to call depending on what the user wants to achieve.

Code:
Class for airport, plane and weather.  Well separated files, and separate test files to match these.
Weather class is well refactored but it would be good to see information in the README about the decision on 1/7 probability of stormy weather.

Airport Code is very well presented and is easy to read and follow.  methods are in a logical order.
Well refactored.

Tests:

Shows good knowledge of rspec.  A before test to set up the test environment is very sensible.  As is the context test to set up stormy weather.
Tests are in a logical order.
Well presented and easy to follow.
Accept from comments that doubles need to be worked on to be able to be included.
Next step - think about edge cases e.g. Can a plane be told to land if it is already in the airport?
Try using Rubocop to help with code presentation (or similar).

8 tests with 100% coverage.  Efficient code!

File: lib/airport.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  def initialize(capacity=DEFAULT_CAPACITY, weather = Weather.new)
+    @plane = []
+    @capacity = capacity
+    @weather = weather
+  end
+
+  def land(plane)
+    fail  'Cannot land the plane, poor weather' if @weather.storm? == :storm
+    fail 'Warning, airport full' if full?
+    @plane << plane
```
Comment (eoinmakers) [inline lib/airport.rb:18]: It would be clearer to differentiate between a plane object being passed in and the array named plane acting as a hangar. Hangar, or planes, is a slightly more descriptive name for the latter.

Comment (eoinmakers) [reply lib/airport.rb:18]: Really cleanly written method!

**Review by eoinmakers — COMMENTED** [review]
Nice work Charlie. :)

**Review by eoinmakers — COMMENTED** [review]
Nice work Charlie. :)

**Review by eoinmakers — COMMENTED** [review]
*duplicate comment*
